src/queries/catalog/call-context-query/call-context-query-format.ts

```diff
@@ -12,26 +12,34 @@ import type { DEFAULT_DATAFLOW_PIPELINE } from '../../../core/steps/pipeline/def
 import { CallTargets } from './identify-link-to-last-call-relation';
 
 export interface DefaultCallContextQueryFormat<CallName extends RegExp | string> extends BaseQueryFormat {
-	readonly type:            'call-context';
+	readonly type:                   'call-context';
 	/** Regex regarding the function name, please note that strings will be interpreted as regular expressions too! */
-	readonly callName:        CallName;
+	readonly callName:               CallName;
 	/**
 	 * Should we automatically add the `^` and `$` anchors to the regex to make it an exact match?
 	 */
-	readonly callNameExact?:  boolean;
+	readonly callNameExact?:         boolean;
 	/** kind may be a step or anything that you attach to the call, this can be used to group calls together (e.g., linking `ggplot` to `visualize`). Defaults to `.` */
-	readonly kind?:           string;
+	readonly kind?:                  string;
 	/** subkinds are used to uniquely identify the respective call type when grouping the output (e.g., the normalized name, linking `ggplot` to `plot`). Defaults to `.` */
-	readonly subkind?:        string;
+	readonly subkind?:               string;
 	/**
 	 * Call targets the function may have. This defaults to {@link CallTargets#Any}.
 	 * Request this specifically to gain all call targets we can resolve.
 	 */
-	readonly callTargets?:    CallTargets;
+	readonly callTargets?:           CallTargets;
 	/**
 	 * Consider a case like `f <- function_of_interest`, do you want uses of `f` to be included in the results?
 	 */
-	readonly includeAliases?: boolean;
+	readonly includeAliases?:        boolean;
+	/**
+	 * Regex that, when set, a node's file attribute must match to be considered
+	 */
+	readonly fileFilter?:            string;
+	/**
+	 * If `fileFilter` is set, but a nodes `file` attribute is `undefined`, should we include it in the results? Defaults to `true`.
```

Comment:
I think it would be nicer to attach the includeUndefinedFiles flag to the `fileFilter`: 

```ts
readonly fileFilter?: {
     readonly name: string;
     readonly includeUndefinedFiles?: boolean;
}
```

What do you think?